acme/acme/crypto_util_test.py

```diff
@@ -151,6 +151,55 @@ def test_sn_collisions(self):
             self.serial_num.append(cert.get_serial_number())
         self.assertTrue(len(set(self.serial_num)) > 1)
 
+class FunctionTest(unittest.TestCase):
+    """Test for standalone functions."""
+
+    def test_make_private_key(self):
+        from acme.crypto_util import make_private_key
+        pem = make_private_key()
+        self.assertTrue(b'--BEGIN PRIVATE KEY--' in pem)
+        self.assertTrue(b'--END PRIVATE KEY--' in pem)
+
+    def test_make_csr(self):
+        from acme.crypto_util import make_private_key, make_csr
```

Comment:
Another common testing convention followed in this repo is to import whatever you're testing in the smallest scope possible. The way I'd do this is to make a method like:

```
@classmethod
_call_with_key(cls, *args, **kwargs):
```
that generates a key, imports `make_csr`, and returns its return value.